modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,106 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'swcom'
+
+@description('Optional. Data Lake Storage Filesystem.')
+param dataLakeStorageFilesystem string = 'synapsews'
```

Comment:
In my opinion this should be hard-coded in the dependency deployment. Similar to the vnet subnet as they don't need to be unique

Reply:
it is needed to create the container and as an input parameter that's why I added it as a parameter